### PR TITLE
Added recipe for pyfiglet

### PR DESCRIPTION
**Pull request by pmlandwehr**
[`pyfiglet`](https://github.com/pwaller/pyfiglet) is a python-implementation of [`FIGlet`](http://www.figlet.org/), which makes it easy to do fancy ASCII text headers.

### CODE REVIEW

File: recipes/pyfiglet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    -  setuptools
```
Comment (jakirkham) [inline recipes/pyfiglet/meta.yaml]: Is this still needed? If so, probably should fix the indenting.

Comment (pmlandwehr) [inline recipes/pyfiglet/meta.yaml]: Yeah, it's needed.

Comment (jakirkham) [inline recipes/pyfiglet/meta.yaml]: Indeed. Am seeing [`pkg_resources`]( https://github.com/pwaller/pyfiglet/blob/v0.7.5/pyfiglet/__init__.py#L10 ).

**Comment by pmlandwehr** [comment]
@jakirkham I think this is good to go.

**Comment by pmlandwehr** [comment]
Can I get a confirmation on this, @conda-forge/staged-recipes?

**Pull request merged**